src/content/docs/react/introduction.mdx

```diff
@@ -1,6 +1,102 @@
 ---
-title: Introduction
+title: Introduction to React
+description: An introduction to react with an overview of it's main features and how to start a new project
 ---
 
-Guides lead a user through a specific task they want to accomplish, often with a sequence of steps.
-Writing a good guide requires thinking about what your users are trying to do.
+import { Card } from '@astrojs/starlight/components';
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import { Code } from '@astrojs/starlight/components';
+import { FileTree } from '@astrojs/starlight/components';
+
+Welcome! This page will educate you about React and how to scaffold your first project with it.
+
+## Why React?
+
+Firstly, what is React? React is an open-source Javascript library created by Facebook (Meta) to make creating user interfaces much more easier, while it is deemed as a _frontend framework_ it isn't a framework technically. React is the most popular frontend technology and boasts the following features:
+
+- **Component based architecture**: Allows you to break down UI into reusable components which makes it incredibly easier to manage the UI of your app. 
+- **Virtual DOM**: React uses a virtual DOM which allows it to efficiently update the UI without having to re-render the entire app. This heavily boosts performance 
+- **Declarative**: React uses a declarative approach to define how your UI should look based on its current state. You specify what you want, and React takes care of updating the DOM to match that desired state.
+- **JSX**: React uses JSX (Javascript Syntax Extension) which makes development much more intutive allowing ES6 Javascript to allow HTML-like syntax to coexist with it.
+- _And much more..._
```

Comment:
Please add "And much more" in the above point itself instead of another separate point.